Haneke/Haneke.swift

```diff
@@ -53,3 +53,13 @@ func errorWithCode(_ code: Int, description: String) -> Error {
     let userInfo = [NSLocalizedDescriptionKey: description]
     return NSError(domain: HanekeGlobals.Domain, code: code, userInfo: userInfo) as Error
 }
+
+internal var defaultDiskCapacityStrategy = deleteItemsOverCapacity
+
+
+/// Set the module wide default disk cache invalidation strategy
+///
+/// - Parameter strategy: new disk cache invalidation strategy
+public func setDefaultDiskCapacityStrategy(_ strategy: @escaping (DiskCache, FileManager) -> Void) {
```

Comment:
Is there any way to scope this to Haneke? Something about this being a **global** function that anything anywhere in the project can call feels a little dangerous.

Reply:
yes but it would feel a little weird. https://medium.com/swift-programming/swift-enums-as-namespace-7df63a17f36f

let me know what you think.

Reply:
I ended up just moving this method into ```Shared```